addons/binding/org.openhab.binding.openweathermap/README.md

```diff
@@ -112,8 +112,8 @@ Once the parameter `forecastDays` will be changed, the available channel groups
 | forecastHours03, forecastHours06, ... forecastHours120 | wind-direction | Number:Angle         | Forecasted wind direction.                                                 |
 | forecastHours03, forecastHours06, ... forecastHours120 | gust-speed     | Number:Speed         | Forecasted gust speed. **Advanced**                                        |
 | forecastHours03, forecastHours06, ... forecastHours120 | cloudiness     | Number:Dimensionless | Forecasted cloudiness.                                                     |
-| forecastHours03, forecastHours06, ... forecastHours120 | rain           | Number:Length        | Expected rain volume for the next 3 hours.                                 |
-| forecastHours03, forecastHours06, ... forecastHours120 | snow           | Number:Length        | Expected snow volume for the next 3 hours.                                 |
+| forecastHours03, forecastHours06, ... forecastHours120 | rain           | Number:Length        | Expected rain volum.                                                      |
```

Comment:
Spelling: `volum`

Reply:
Ok, fixed that one. Must have been slipped through, as the white space was also missing.